translations/parkour/id.lua

```diff
@@ -3,51 +3,51 @@ translations.id = {
     fullname = "Bahasa Indonesia",
 
     -- Error messages
-    corrupt_map = "<r>Peta rusak. Sedang Memuat lainnya.",
+    corrupt_map = "<r>Peta rusak. Sedang Memuat peta lainnya.",
     corrupt_map_vanilla = "<r>[KESALAHAN] <n>Tidak bisa mendapatkan informasi dari peta ini.",
     corrupt_map_mouse_start = "<r>[KESALAHAN] <n>Peta ini harus memiliki posisi awal (titik spawn tikus).",
     corrupt_map_needing_chair = "<r>[KESALAHAN] <n>Peta harus memiliki kursi di akhir.",
     corrupt_map_missing_checkpoints = "<r>[KESALAHAN] <n>Peta harus memiliki setidaknya satu cekpoin (paku kuning).",
-    corrupt_data = "<r>Sayangnya, data kamu rusak dan telah disetel ulang.",
-    min_players = "<r>Untuk menyimpan data kamu, setidaknya ada 4 pemain di ruangan ini. <bl>[%s/%s]",
+    corrupt_data = "<r>Sayangnya, data anda rusak dan telah disetel ulang.",
+    min_players = "<r>Untuk menyimpan data anda, setidaknya ada 4 pemain di ruangan ini. <bl>[%s/%s]",
```

Comment:
'4 pemain' can be changed to '4 pemain unik' because one user can play more than one account.

Reply:
have other words besides **unik**?

Reply:
hmm i don't have any word yet for translate unique to Bahasa Indonesia.